python-sdk/pachyderm_sdk/client.py

```diff
@@ -229,7 +230,7 @@ def from_config(cls, config_file: Union[Path, str]) -> "Client":
         """
         # TODO: Should config_file be nullable?
         #  If null should we search for the local config?
-        config = _ConfigFile(config_file)
+        config = ConfigFile(config_file)
```

Comment:
Can't comment on exact line, but it might be worth moving the `env_config`, `spout_config` and `local_config` class variables in this PR. Maybe move them to some combination of `constants.py` and the new `config.py`?